src/components/UI/OnlineIndicator.vue

```diff
@@ -1,48 +1,58 @@
 <template>
-  <span :class="$style.indicator" :data-is-online="isOnline" :title="tooltip" />
+  <indicator
+    :class="$style.container"
+    :data-is-online="isOnline"
+    :size="size"
+    :border-width="isOnline ? 0 : borderWidth"
+  />
 </template>
 
 <script lang="ts">
 import { defineComponent, computed, PropType } from '@vue/composition-api'
 import store from '@/store'
 import { UserId } from '@/types/entity-ids'
-import { getFullDayWithTimeString } from '@/lib/date'
+import Indicator from './Indicator.vue'
 
 export default defineComponent({
   name: 'OnlineIndicator',
+  components: {
+    Indicator
+  },
   props: {
     userId: {
       type: String as PropType<UserId>,
       required: true
     },
-    lastOnline: String
+    size: {
+      type: Number,
+      default: 14
```

Comment:
ボーダーとのサイズ感の兼ね合い（あと歪みが治ったら嬉しい）
```suggestion
      default: 12
```

Reply:
なんか`border-radius: 100vw`が怪しかったっぽいです